resources/kcp/charts/keda/README.md

```diff
@@ -0,0 +1,147 @@
+</p>
+
+---
+
+## TL;DR
+
+```console
+helm repo add kedacore https://kedacore.github.io/charts
+helm repo update
+
+kubectl create namespace keda
+helm install keda kedacore/keda --namespace keda
+```
+
+## Introduction
+
+This chart bootstraps KEDA infrastructure on a Kubernetes cluster using the Helm package manager.
+
+As part of that, it will install all the required Custom Resource Definitions (CRD).
+
+## Installing the Chart
+
+To install the chart with the release name `keda`:
+
+```console
+$ kubectl create namespace keda
+$ helm install keda kedacore/keda --namespace keda
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `keda` Helm chart:
+
+```console
+helm uninstall keda
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the Promitor chart and
+their default values.
+
+| Parameter                                                  | Description                               | Default                                         |
+|:-----------------------------------------------------------|:------------------------------------------|:------------------------------------------------|
+| `image.keda.repository`                                    | Image name of KEDA operator               | `ghcr.io/kedacore/keda`                |
+| `image.keda.tag`                                           | Image tag of KEDA operator. Optional, given app version of Helm chart is used by default | `` |
+| `image.metricsApiServer.repository`                        | Image name of KEDA Metrics API Server        | `ghcr.io/kedacore/keda-metrics-apiserver` |
+| `image.metricsApiServer.tag`                               | Image tag of KEDA Metrics API Server. Optional, given app version of Helm chart is used by default | `` |
+| `crds.install`                               | Defines whether the KEDA CRDs have to be installed or not. | `true`                 |
+| `watchNamespace`                                           | Defines Kubernetes namespaces to watch to scale their workloads. Default watches all namespaces | `` |
+| `operator.name`                                            | Name of the KEDA operator | `keda-operator` |
+| `operator.replicaCount`                                      | Capability to configure the number of replicas for KEDA operator.<br /><br />While you can run more replicas of our operator, only one operator instance will be the leader and serving traffic.<br /><br />You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce downtime during a failover.| `1` |
+| `metricsServer.dnsPolicy`                                  | Defined the DNS policy for the metric server | `ClusterFirst`
+| `metricsServer.useHostNetwork`                             | Enable metric server to use host network  | `false`

```

Comment:
```suggestion
| `metricsServer.useHostNetwork`                             | Enables the metric server to use host network  | `false`

```

Reply:
See first comment. This was resolved after our Slack discussion.